pkg/controller.v1/pytorch/job.go

```diff
@@ -148,7 +149,7 @@ func (pc *PyTorchController) updatePyTorchJob(old, cur interface{}) {
 	}
 }
 
-func (pc *PyTorchController) deletePodsAndServices(job *pyv1.PyTorchJob, pods []*v1.Pod) error {
+func (pc *PyTorchController) deletePodsAndServices(job *pyv1.PyTorchJob, pods []*v1.Pod,services []*v1.Service) error {
```

Comment:
I think we should have some comments to illustrate the behavior of the function.

Reply:
How about to use “delete all the pods and master servics”

Reply:
SGTM

Reply:
```suggestion
// deletePodsAndServices deletes all the pods and master service.
func (pc *PyTorchController) deletePodsAndServices(job *pyv1.PyTorchJob, pods []*v1.Pod,services []*v1.Service) error {
```

Reply:
LGTM. Keeping the function name in the comment is sort of golang convention.

Reply:
Great